master/getting-started/bare-metal/bare-metal.md

```diff
@@ -31,21 +31,22 @@ However, for host endpoints, Calico is more lenient; it only polices
 traffic to/from interfaces that it's been explicitly told about. Traffic
 to/from other interfaces is left alone.
 
-As of Calico v2.1.0, Calico applies host endpoint security policy both to traffic
-that is terminated locally, and to traffic that is forwarded between host
-endpoints.  Previously, policy was only applied to traffic that was terminated
-locally.  The change allows Calico to be used to secure a NAT gateway or router.
-Calico supports selector-based policy as normal when running on a gateway or router
-allowing for rich, dynamic security policy based on the labels attached to your
-workloads.
-
-> **Note**: If you have a host with workloads on it then traffic that is forwarded to
-> workloads bypasses the policy applied to host endpoints. If that weren't the
-> case, the host endpoint policy would need to be very broad to allow all
-> traffic destined for any possible workload.
->
-> Since version 2.1.0, Calico applies host endpoint policy to traffic that is
-> being forwarded between host interfaces.
+You can use host endpoint policy to secure a NAT gateway or router. Calico
+supports selector-based policy when running on a gateway or router, allowing for
+rich, dynamic security policy based on the labels attached to your host endpoints. 
+
+You can apply host endpoint policies to three types of traffic:
+- Traffic that is terminated locally.
+- Traffic that is forwarded between host endpoints.
+- Traffic that is forwarded between host endpoint and workload endpoint on the 
+same host.
+
+Set the `applyOnForward` flag to `true` to apply a policy to forwarded traffic.
+See [policy spec]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/policy#spec).
+
+> **Note**: Both traffic forwarded between host endpoints and traffic forwarded 
+> between host endpoint and workload endpoint on the same host is regarded as
```

Comment:
Also missing a couple of articles here, I believe. Not a huge matter, but it does make the sentence incorrect, grammatically.